recipes/nf-test/meta.yaml

```diff
@@ -1,17 +1,17 @@
-{% set version = "0.6.0" %}
+{% set version = "0.6.1" %}
 
 
 package:
   name: "nf-test"
   version: {{ version }}
 
 build:
-  number: 0
```

Comment:
Build nr needs to be zero on new versions

Reply:
Thanks! I read it wrong in the tutorial, my bad :)